src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -398,5 +408,17 @@ public static void ValidateCredentials(string username, string password, string
                 throw new CommandException(Strings.SourcesCommandCredentialsRequiredWithAuthTypes);
             }
         }
+
+        public static int ParseProtocolVersion(string protocolVersion)
```

Comment:
Please change the access modifier to `private` instead of `public` because this method is used in the same class.

Reply:
This method cannot be made private as it is in class [`internal static class RunnerHelper`](https://github.com/NuGet/NuGet.Client/blob/34f2638d7ebe5d9f4a33f74e9b900c0abad19437/src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs#LL325C39-L325C39) and is used in `AddSourceRunner` and `UpdateSourceRunner` classes.
It was marked `public static` similarly to other methods in this class.

Reply:
@kartheekp-ms this conversation is not marked as resolved. Can you please check if you have any other comments?